app/controllers/EventDetailsController.js

```diff
@@ -0,0 +1,10 @@
+GameUpApp.controller('EventDetailsController', ['$scope', '$location', 'eventService', 'events', function($scope, $location, eventService, events){
+  $scope.eventObj = eventService.getEventId();
+
+  $scope.formatDate = function(date) {
```

Comment:
This function is duplicated in app/controllers/ListEventsController.js lines 16-20. Possible to avoid this duplicated code?